docs/spec/spec.md

```diff
@@ -273,6 +273,8 @@ The GraphQL schema is generated by analyzing the Ballerina service attached to t
 
 When an incompatible type is used inside a GraphQL service, a compilation error will be thrown.
 
+**NOTE:** The generated GraphQL schema is exported by the GraphQL package in Schema Definition Language(SDL). For the Ballerina projects, the schema file is written into `target` directory and for the single bal files, the schema is written into the same directory where the bal file is located.
+
```

Comment:
Let's reorganize this to make it a part of the description, not as a separate note.